actions/gha-workflow-validator/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import { commentOnPrOrUpdateExisting, deleteCommentOnPRIfExists, getComparison } from "./github.js";
+import { validateActionReferenceChanges } from "./action-reference-validations.js";
+import { filterForGithubWorkflowChanges, formatGithubComment, parseAllAdditions } from "./utils.js";
+import { COMMENT_HEADER } from "./strings.js";
+
+(async () => {
+  const { token, owner, repo, base, head, prNumber } = getInvokeContext();
+  const octokit = github.getOctokit(token);
+
+  const allFiles = await getComparison(octokit, owner, repo, base, head);
+  const ghaWorkflowFiles = filterForGithubWorkflowChanges(allFiles);
+  const ghaWorkflowPatchAdditions = parseAllAdditions(ghaWorkflowFiles);
+
+  const containsWorkflowModifications = ghaWorkflowPatchAdditions.some(file => {
+    return (file.filename.startsWith('.github/workflows') || file.filename.startsWith('.github/actions')) && (file.filename.endsWith('.yml') || file.filename.endsWith('.yaml'))
+  });
+
+  if (containsWorkflowModifications) {
```

Comment:
```suggestion
  if (!containsWorkflowModifications) return
```
Would be good if we can reduce nesting where we can

Reply:
Unnested